threadsd/main.go

```diff
@@ -2,11 +2,12 @@ package main
 
 import (
 	"context"
-	"flag"
 	"fmt"
+	"os"
 
 	logging "github.com/ipfs/go-log"
 	ma "github.com/multiformats/go-multiaddr"
+	"github.com/namsral/flag"
```

Comment:
This is a drop in replacement for `flag` that looks to env vars after command line args.